### ISSUE TITLE
In-Home Streaming

### ISSUE THREAD

**Issue by karasuhebi** [xan105/FFX-X2_LAUNCHER#14]
This launcher doesn't seem to play nice with In-Home Streaming. I've tried it with two different host PCs and both of them results in a black screen.

I can hear the launcher's music and I can see the mouse cursor, but the screen is otherwise completely black.

**Comment by xan105** [comment]
Maybe it's because of the gpu acceleration, but i have no way to test in-home-streaming.
Would you be so kind as to test with this one ?
https://github.com/xan105/FFX-X2_LAUNCHER/releases/download/untagged-613aa769c158c2cb15e7/FFX.X-2_LAUNCHER.7z

**Comment by karasuhebi** [comment]
Yeah, I can test it. The link is broken though. Have another one? :)

It's not GPU acceleration though, I have it turned off on the host side:
https://i.imgur.com/Kn9H0My.png

**Comment by xan105** [comment]
I meant the gpu acceleration inside my app not steam :)
Here is another link : https://mega.nz/#!wp9HXBLA!ukUs-IuhnY4GrNvxUIVJGdyyhciDEZ2W-2KrgO_qwB0

**Comment by Samutz** [comment]
I tested this today as I was also getting a black screen. It fixes the black screen, but the launcher doesn't respond to my Xbox One controller connected to my Steam Link. I've only been able to get it to respond to a controller connected directly to the PC when launching from the PC. Both controllers work fine in the actual game though.

The same is true if I use the latest release of the launcher. If I go look at it on the PC (since it's black on Steam Link) it doesn't respond to the controller.
